lib/cadet_web/views/sourcecast_view.ex

```diff
@@ -16,7 +16,8 @@ defmodule CadetWeb.SourcecastView do
       audio: :audio,
       playbackData: :playbackData,
       uploader: &transform_map_for_view(&1.uploader, [:name, :id]),
-      url: &Cadet.Courses.SourcecastUpload.url({&1.audio, &1})
+      url: &Cadet.Courses.SourcecastUpload.url({&1.audio, &1}),
+      course_id: :course_id
```

Comment:
`courseId`

TBH, I would rather the backend stop doing this snake case <=> camel case conversion nonsense, and just have snake case in the frontend, but ... oh well